### ISSUE TITLE
Upgrade to 8.1.3 unsuccessful and now no access to data

### ISSUE THREAD

**Issue by jbb0906** [owncloud/core#19190]
I just tried updating to 8.1.3 from 8.1.0 using the update app under administration.  I received the following error.

![owncloud update fail](https://cloud.githubusercontent.com/assets/1658332/9976463/4c5e795e-5edd-11e5-8132-4b357a1179ac.jpg)

Please help. My technical knowledge and skills are limited, so please explain in plain English, what I can do to regain access to my owncloud install.

**Comment by Xenopathic** [comment]
Run the upgrade a second time, it will work the second time.

**Comment by Xenopathic** [comment]
Marking as duplicate of #19145
